HACKING.rst

```diff
@@ -0,0 +1,32 @@
+NAPALM Style Commandments
+=======================
+
+- Step 1: Read the OpenStack Hacking Standards
+  `here <http://docs.openstack.org/developer/hacking/>`_.
+- Step 2: Read on
+
+NAPALM Specific Commandments
+--------------------------
+
+- [N319] Validate that debug level logs are not translated
+- [N320] Validate that LOG messages, except debug ones, have translations
```

Comment:
It's part of the openstack rules, specifically around internationalization. We can choose to follow or not follow this, but in this context it means using libraries like `oslo.i18n` (see [here](https://wiki.openstack.org/wiki/Translations#Internationalization_.28i18n.29))